scene/animation/tween.cpp

```diff
@@ -1185,44 +1185,38 @@ bool Tween::_build_interpolation(InterpolateType p_interpolation_type, Object *p
 	// Validate and apply interpolation data
 
 	// Give it the object
-	ERR_EXPLAIN("Invalid object provided to Tween!");
-	ERR_FAIL_COND_V(p_object == NULL, false); // Is the object real
-	ERR_FAIL_COND_V(!ObjectDB::instance_validate(p_object), false); // Is the object a valid instance?
+	ERR_FAIL_COND_V_MSG(p_object == NULL, false, "Invalid object provided to Tween."); // Is the object real
+	ERR_FAIL_COND_V_MSG(!ObjectDB::instance_validate(p_object), false, "Invalid object provided to Tween."); // Is the object a valid instance?
 	data.id = p_object->get_instance_id();
 
 	// Validate the initial and final values
-	ERR_EXPLAIN("Initial value type does not match final value type!"); // TODO: Print both types to make debugging easier
-	ERR_FAIL_COND_V(p_initial_val.get_type() != p_final_val.get_type(), false); // Do the initial and final value types match?
+	// TODO: Print both types to make debugging easier
+	ERR_FAIL_COND_V_MSG(p_initial_val.get_type() != p_final_val.get_type(), false, "Initial value type does not match final value type."); // Do the initial and final value types match?
```

Comment:
Let's solve the TODO:
```
"Initial value type '" + Variant::get_type_name(p_initial_val.get_type()) + "' does not match final value type '" + Variant::get_type_name(p_final_val.get_type()) + "'."
```
(Untested, but if it compiles it should be fine I guess.)

Reply:
The trailing comments in this class are a bit useless BTW, they just state exactly what the test + explanatory message say... but well, it's unrelated to this PR :P

Reply:
I was going to solve it, but wasn't sure how to get the type string.
(also removed the comments xd)